ui/feed_list.go

```diff
@@ -26,6 +26,11 @@ func (h *handler) showFeedsPage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	feedCountUnread := make(map[int64]int)
+	for _, feed := range feeds {
+		feedCountUnread[feed.ID] = h.store.CountUnreadFeedEntries(user.ID, feed.ID)
```

Comment:
That will generates lot of SQL queries for those with lot of feeds. There is probably a way of doing a subquery instead of looping through all feeds.

Reply:
it sure does, I'll look into it 👍

Reply:
I've changed the code to run a single query to retrieve all counters per feed, let me know what you think!